templates/bareos-fd/director/bareos-dir.conf.j2

```diff
@@ -1,5 +0,0 @@
-Director {
-  Name = bareos-dir
-  Password = "{{ bareos_client_password  }}"
-  Description = "Allow the configured Director to access this file daemon."
```

Comment:
There is no description in the other template. Also, the format of the password is different:
https://github.com/mila-iqia/ansible-role-bareos/blob/dfaeecba650c953417f3caa7ad09d04c6f5a2652/templates/bareos-dir.conf.j2#L1-L5

Can this break something?

Reply:
Nope, nothing can break. We simply used the cleartext password instead of the md5 version, but both work.

Reply:
Does one need to update their inventory after this change?

Reply:
No.